test/archetype.test.ts

```diff
@@ -2064,6 +2064,58 @@ describe("Factory", function () {
 
     await expect(await nftMint.totalSupply()).to.be.equal(2);
   });
+
+  it("test erc115 test batch mint verification", async function () {
+    const [accountZero, accountOne, accountTwo] = await ethers.getSigners();
+    let default_config = { ...DEFAULT_CONFIG };
+    default_config.maxSupply = new Array(500).fill(10); // 500 tokenIds, 10 each
+    default_config.maxBatchSize = 1000;
```

Comment:
eslint flags this 
<img width="511" alt="image" src="https://user-images.githubusercontent.com/87461192/230746594-236723af-bef4-4e1a-95f8-f1c839dfcdfe.png">

Use 
```
    const default_config = {
      ...DEFAULT_CONFIG,
      maxSupply: new Array(500).fill(10),
      maxBatchSize: 1000,
    };
 ```